### DISCUSSION TITLE
RFM98PW power configuration

### DISCUSSION THREAD

**Discussion by Sumya99** [jgromes/RadioLib#704] [Q&A] [answered]
Hello sir, 

I have been using RFM98PW module and its datasheet specifies that it can generate output power until +30dBm. Excepting hardware conceptions, is it possible to make it work with 30dBm output power using RadioLib? Or should I change the limit power ranges ? 

Thanks in advance :)

**Comment by jgromes** [comment, marked as answer]
Take a look at [this wiki page](https://github.com/jgromes/RadioLib/wiki/High-power-Radio-Modules-Guide), which shows how to use high-power modules in RadioLib.

**Reply by Sumya99** [reply]
Thank you for the reply. Is current limit register only about transceiver IC's current consumption or the whole module?

**Reply by jgromes** [reply]
The current limit is internal to the transceiver IC only, it's a functionality of the IC.
